eth/protocols/eth/handlers.go

```diff
@@ -190,6 +191,11 @@ func AnswerGetReceiptsQuery(br services.FullBlockReader, db kv.Tx, query GetRece
 				continue
 			}
 		}
+		err = AddLogsToReceipts(db, results, b, exec)
```

Comment:
`rawdb.ReadReceipts` can't use this method - it doesn't work in E3 (doesn't store receipts in db).